src/table/table.scss

```diff
@@ -16,75 +15,205 @@
     box-sizing: border-box;
   }
 
-  > .iui-tables-body {
-    width: 100%;
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
+  .iui-table-header {
+    @include iui-table-header;
+  }
+
+  .iui-table-body {
+    @include iui-table-body;
+  }
+
+  .iui-cell {
+    @include iui-table-cell;
   }
 
-  &.iui-zebra-striping > .iui-tables-body > .iui-tables-row:nth-child(even) {
+  &.iui-zebra-striping > .iui-table-body > .iui-row:nth-child(even):not(.iui-selected) {
     @include themed {
       background: rgba(t(iui-color-foreground-body-rgb), 0.02);
     }
   }
 
-  &.iui-tables-fixed > .iui-tables-body {
-    max-height: 300px;
+  &.iui-fixed > .iui-table-body {
+    max-height: $iui-baseline * 27;
     overflow-x: hidden;
     overflow-y: scroll;
     overflow: overlay;
   }
+}
 
-  .iui-select .iui-select-button {
-    background-color: transparent;
-    border: none;
+@mixin iui-table-header {
+  width: 100%;
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
   }
 
-  .iui-tables-row {
-    margin-top: -1px;
+  .iui-row {
     display: flex;
     flex-grow: 1;
     width: 100%;
-    border-top: 1px solid transparent;
-    border-left: 1px solid transparent;
-    border-right: 1px solid transparent;
+  }
+
+  .iui-cell:not(.iui-slot) {
+    @include iui-focus;
+    cursor: pointer;
+    position: relative;
+    overflow: visible;
+
+    > .iui-borderless {
+      margin-left: $iui-xs;
+
+      &:not(.iui-active) {
+        visibility: hidden;
+      }
+    }
+
+    > .iui-resizer {
+      height: 100%;
+      width: $iui-m;
+      position: absolute;
+      top: 0;
+      right: 0;
+      transform: translateX(50%);
+      touch-action: none;
+      cursor: ew-resize;
+      z-index: 100;
+      display: none;
+        
+      > .iui-resizer-bar {
+        height: 100%;
+        width: $iui-xxs;
+        margin: 0 auto;
+        transition: background-color $iui-speed-fast ease-out, width $iui-speed-fast ease-out;
+        @include themed {
+          background-color: t(iui-color-background-5);
+        }
+      }
+
+      &:hover > .iui-resizer-bar {
+        width: $iui-xs;
+        @include themed {
+          background-color: t(iui-color-foreground-primary);
+        }
+      }
+    }
+
+    &:hover,
+    &:focus,
+    &:focus-within {
+      @include themed {
+        background-color: t(iui-color-background-4);
+      }
 
+      .iui-sort,
+      .iui-borderless {
+        visibility: visible;
+      }
+    }
+
+    &:hover {
+      > .iui-resizer {
+        display: block;
```

Comment:
Could you move this up in line 93 together with hover rule for iui-resizer-bar?
Would be better readability...

Reply:
I can move this up to line 98 (line count may have shifted in my local copy from other unrelated changes).  The `.iui-resizer` is being set to block when hovering over `.iui-cell`, not when hovering over `.iui-resizer`.  Line 93 has to do with hovering over the resizer itself, not the cell.